src/test/java/com/paypal/base/BaseTest.java

```diff
@@ -22,6 +23,7 @@ public class BaseTest extends EnvironmentSetter {
     protected String browser;
     protected String browserVersion;
     protected LoginPage loginPage;
+    protected String downloadsDirectoryPath = "C:/Users/Amardeep/Downloads";
```

Comment:
avoid path dependency. Instead use system.getProperty("user.dir")

Reply:
Merci Dave. I will use that.